data/hails.txt

```diff
@@ -680,9 +679,9 @@ phrase "friendly civilian"
 		"smashed the"
 		"walloped the"
 		"lose to the"
-		"get obliterated by"
-		"lose pathetically to the"
-		"have no chance against the"
+		"to get obliterated by"
```

Comment:
"The Void Dragons to get obliterated by Jabberwocks!"

Reply:
"to be obliterated by the" is better?

Reply:
Say this sentence out loud: "The Void Dragons to get obliterated by Jabberwocks!"

it's simply not correct English. The string needs to be changed to something like, "got obliterated by the"

Reply:
ty, I don't know why I couldn't understand what you meant till now. changed it, haven't commited it yet tho.